python/docs/source/reference/pyspark.pandas/resampling.rst

```diff
@@ -0,0 +1,42 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+
+.. _api.resampling:
+
+==========
+Resampling
```

Comment:
(should add this page to https://github.com/apache/spark/blob/253627e4dbb62af9f94d35ee3140024754a942e9/python/docs/source/reference/pyspark.pandas/index.rst)